frontend/web/micro-ui-internals/packages/ui-components/src/atoms/BreadCrumb.js

```
@@ -0,0 +1,134 @@
+import React, { useEffect, useState, Fragment } from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+const BreadCrumb = (props) => {
+  const [expanded, setExpanded] = useState(false);
+  const [crumbsToDisplay, setCrumbsToDisplay] = useState([...props?.crumbs]);
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)